bin/agent-data-plane/src/config.rs

```diff
@@ -19,6 +19,10 @@ pub enum Action {
     /// Various debugging commands.
     #[command(subcommand)]
     Debug(DebugConfig),
+
+    /// Prints the version of the agent data plane.
+    #[command(name = "dogstatsd-stats")]
+    DogstatsdStats,
```

Comment:
We should make this a subcommand under `dogstatsd`.

We'll eventually end up having other DSD-specific things (capturing the raw payloads, etc), so it'll look a little nicer as proper subcommands, and we don't need to _exactly_ match how it's done in the Core Agent since users are going to have to type `agent-data-plane ...` instead of `agent ...` to begin with.

Reply:
Sorry, missed your response...

Yes, we want there to be a `dogstatsd` subcommand group, with this PR adding a `statsd` subcommand in that group.

Reply:
Sorry: `stats`, not `statsd`. 😅